tests/cases/draw_rect_multiple_color_attachemnt.amber

```diff
@@ -0,0 +1,40 @@
+#!amber
+# Copyright 2019 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHADER vertex vert_shader PASSTHROUGH
+SHADER fragment frag_shader GLSL
+#version 430
+layout(location = 0) out vec4 color_out;
+layout(location = 1) out vec4 color_out1;
+void main() {
+  color_out = vec4(0.0, 0.5, 0.0, 0.8);
+  color_out1 = vec4(1.0, 1.0, 1.0, 1.0);
+}
+END
+
+BUFFER framebuffer FORMAT B8G8R8A8_UNORM
+BUFFER framebuffer1 FORMAT B8G8R8A8_UNORM
+
+PIPELINE graphics my_pipeline
+  ATTACH vert_shader
+  ATTACH frag_shader
+  FRAMEBUFFER_SIZE 800 600
+  BIND BUFFER framebuffer AS color LOCATION 0
+  BIND BUFFER framebuffer1 AS color LOCATION 1
+END
+
+RUN my_pipeline DRAW_RECT POS 0 0 SIZE 250 250
```

Comment:
I would increase this to SIZE 800 600 since you set the framebuffer size to that.